ARE/components/processor.serialport/src/main/java/eu/asterics/component/processor/serialport/SerialPortInstance.java

```diff
@@ -209,7 +214,20 @@ public Object setRuntimePropertyValue(String propertyName, Object newValue) {
             propSendBytesBufferSize = Integer.parseInt(newValue.toString());
             sendBytesBuffer=ByteBuffer.allocate(propSendBytesBufferSize);
             return oldValue;
-        }        
+        }
+        if ("cimId".equalsIgnoreCase(propertyName)) {
+            final Object oldValue = propCimId;
+            String s = newValue.toString();
+            if(s.contains("0x")) {
```

Comment:
maybe a stricter parsing would be: 

if(newValue != null) {
newValue=newValue.trim();
if(newValue.startsWith("0x")....)